scenarios/web_navigation/dataset.py

```
@@ -1,86 +1,202 @@
+        difficulty = task.get("difficulty", "medium")
+        
+        # Try to parse JSON actions if present
+        actions = []
+        if '```json' in response:
+            json_start = response.find('```json') + 7
+            json_end = response.find('```', json_start)
+            json_content = response[json_start:json_end]
+            try:
+                parsed = json.loads(json_content)
+                actions = parsed.get('actions', [])
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)